.swiftlint.yml

```diff
@@ -0,0 +1,8 @@
+disabled_rules:
+  - opening_brace
+  - trailing_comma
```

Comment:
Instead of disabling it, you can actually _require_ the comma by adding this at the top level:

```
trailing_comma:
  mandatory_comma: true
```

Would you mind doing that instead? Sorry that wasn't clear before. 😅